packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -169,6 +170,14 @@ abstract class FlutterDriver {
   /// Getter of serviceClient.
   VMServiceClient get serviceClient => throw UnimplementedError();
 
+  /// Getter of webDriver.
+  async_io.WebDriver get webDriver => throw UnimplementedError();
```

Comment:
Shall we add a message to the error here as well? Also a comment on why we add this method. (Sorry the serviceClient also doesn't have it, but I thought it might be useful in the future to the readers of this code)

Reply:
See the comment below.